test/routes/service-complete.route.test.js

```diff
@@ -738,6 +739,8 @@ const _createSection10RedisMock = (
       : null
   }
 
+  redisMockDataMap[RedisKeys.DESCRIBE_THE_ITEM] = itemDescription
```

Comment:
This value is not actually being checked for in any of the tests, they need to be amended to ensure that that notification service is being called with the correct value

Reply:
Done.